app/routes/wow.ilvl-shopping-list.tsx

```
@@ -277,9 +291,11 @@ const IlvlShoppingListComponent = () => {
           onChange={(e) => setDesiredMinIlvl(e.currentTarget.value)}
         />
         <div className="flex flex-col gap-2">
-          <label className="text-sm font-medium dark:text-gray-200">Desired Stats</label>
+          <label className="text-sm font-medium dark:text-gray-200">
+            Desired Stats
+          </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)